### PR TITLE
Feature/manage slot fe automation

### PR DESCRIPTION
**Pull request by Prajakta1939**
TypeScript frontend test cases.

### CODE REVIEW

File: go1percentTS/src/page-objects/manageSlot/manageSlotPage.ts [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+        slotStartTimeInput: 'input[placeholder="HH"]',
+        successfullyCreatedSlotMessage: 'div[aria-label="Slot Created Successfully"]',
+        cancelButton: 'button[class="btn btn-primary button cancel-button ml-3 mr-2"]',
+        errorMessage: {
+          selector: "//span[contains(text(),'Please do not enter past time')]",
+          locateStrategy: 'xpath',
+        },
+        downArrow:
+        {
+          selector: "(//div[@class='ngx-timepicker-control']//span[@class='ngx-timepicker-control__arrow'])[2]",
```
Comment (Ritika-gupta20) [inline go1percentTS/src/page-objects/manageSlot/manageSlotPage.ts:252]: css loactors can be used here please change

File: go1percentTS/src/page-objects/manageSlot/manageSlotPage.ts [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+      CancelButton: {
+        selector: "//div//button[contains(text(),'Cancel')]",
+        locateStrategy: 'xpath',
+      },
+      deleteButton: {
+        selector: "//button[contains(text(),' Delete')]",
+        locateStrategy: 'xpath',
+      },
+      deletefreeSlot: {
+        selector: "//td//div[contains(text(),'10:40 am')]",
```
Comment (Ritika-gupta20) [inline go1percentTS/src/page-objects/manageSlot/manageSlotPage.ts:284]: same goes here

File: go1percentTS/src/page-objects/manageSlot/manageSlotPage.ts (hunk 1 above)
Comment (Prajakta1939) [reply go1percentTS/src/page-objects/manageSlot/manageSlotPage.ts:252]: ok

File: go1percentTS/src/page-objects/manageSlot/manageSlotPage.ts (hunk 2 above)
Comment (Prajakta1939) [reply go1percentTS/src/page-objects/manageSlot/manageSlotPage.ts:284]: Css is not working here.

**Review by Ritika-gupta20 — APPROVED** [review]

File: go1percentTS/src/page-objects/manageSlot/manageSlotPage.ts (hunk 1 above)
Comment (Prajakta1939) [reply go1percentTS/src/page-objects/manageSlot/manageSlotPage.ts:252]: CSS is not working for that element.

File: go1percentTS/src/test/Go1PercentFEAutomation/manageSlot/manageSlot.ts [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+        it('Verify that create a slot popup should display current date and time', async function (browser: NightwatchBrowser) {
+            manageslotpage
+                .waitForPageLoad()
+                .clickOnDateInCalendar();
+            const currentDate = new Date();
+            const day = currentDate.getDate().toString().padStart(2, '0');
+            const month = (currentDate.getMonth() + 1).toString().padStart(2, '0');
+            const year = currentDate.getFullYear();
+
+            const formattedDate = `${year}-${month}-${day}`;
```
Comment (deepansh257) [inline go1percentTS/src/test/Go1PercentFEAutomation/manageSlot/manageSlot.ts:63]: Can we please create a method of this in page object file and call from there?

**Review by deepansh257 — APPROVED** [review]

File: go1percentTS/src/test/Go1PercentFEAutomation/manageSlot/manageSlot.ts (hunk 3 above)
Comment (Prajakta1939) [reply go1percentTS/src/test/Go1PercentFEAutomation/manageSlot/manageSlot.ts:63]: ok I will try and made changes accordingly.

Comment (Prajakta1939) [reply go1percentTS/src/test/Go1PercentFEAutomation/manageSlot/manageSlot.ts]: Done.

**Review by Ritika-gupta20 — APPROVED** [review]

**Pull request merged**